programmers/이상한-문자-만들기.js

```diff
@@ -1,12 +1,12 @@
 function upperToLowerLowerToUpper(s) {
-  return s.split('').map((val, idx) => {
+  return [...s].map((val, idx) => {
     if (idx % 2) return val.toLowerCase();
     else return val.toUpperCase();
   }).join('');
 }
```

Comment:
if 에서 바로 ```return```을 해주므로
else 구문이 없어도 된답니다.
```suggestion
  return [...s].map((el, idx) => {
    if (idx % 2) return el.toLowerCase();
    return el.toUpperCase();
  }).join('');
}
```